openmdao/api.py

```diff
@@ -95,6 +95,9 @@
 # Units
 from openmdao.utils.units import convert_units, unit_conversion
 
+# Docs Util
+from openmdao.docs.jb_utils import display_source, show_options_table, cite
+
```

Comment:
seems like we don't need these in the `api` module.. these aren't really user-facing?

Reply:
They are technically user facing since contributors can use that to display source code, options, etc

Reply:
I would consider contributors to be developers rather than users..

Reply:
The other reason I did it this way was that at the top of every notebook we `import openmdao.api as om` instead of just `import openmdao` since this is in a separate repo and users opening these books in Colab don't automatically have the full import. I don't think @robfalck wants to add the full import just to access those func so that's why I put the funcs in the API